frontend/src/pages/detail/GenreDetail.tsx

```diff
@@ -0,0 +1,79 @@
+import React, { FormEvent } from "react";
+import { Button } from "primereact/button";
+import { InputText } from "primereact/inputtext";
+import { ToggleButton } from "primereact/togglebutton";
+import ConfirmButton from "../../components/ConfirmButton";
+
+interface modifyGenreState {
+  genre: string;
+  checked: boolean;
```

Comment:
The names of these "checked and "confirmationPopup" booleans aren't very clear to me, I think something like "isModifiable" and "isConfirmationPopupVisible" would be much clearer. This goes for most boolean variables that indicate something; adding an "is" first with a description after will make it much easier to identify what it does